scaffold/nightwatch/docker-compose.selenium.yaml

```diff
@@ -3,7 +3,7 @@ version: '3.6'
 services:
   chrome:
     container_name: ddev-${DDEV_SITENAME}-chrome
-    image: seleniarm/standalone-chromium:4.1.2-20220227
+    image: seleniarm/standalone-chromium:4.5.0-20221001
```

Comment:
🤔 I think renovate supports docker-compose files, I wonder if it would detect this for us.

Reply:
good idea! #60 60